vyxal/elements.py

```diff
@@ -416,12 +416,12 @@ def boolify(lhs, ctx):
     (any) -> is truthy?
     """
     if vy_type(lhs, simple=True) is list:
-        if ctx.truthy_lists:
-            return any_true(lhs, ctx)
-        else:
+        if ctx.vectorise_boolify:
             return vectorise(boolify, lhs, ctx=ctx)
-    else:
-        return int(bool(lhs))
+        else:
+            return any_true(lhs, ctx)
```

Comment:
Wait, are we not doing `int(bool(lhs))` here too?

Reply:
Forgot to change it, but what is better? Before the changes, the `t` flag checked if `any_true(list)`. Is the python version better?

Reply:
@cgccuser changed it